### PR TITLE
[Native] Add test for -Xexport-kdoc

### PR DESCRIPTION
**Pull request by knebekaizer**

### CODE REVIEW

File: kotlin-native/backend.native/tests/objcexport/expectedLazy.h [hunk 1]
```diff
@@ -125,8 +125,14 @@ __attribute__((swift_name("CoroutineException")))
 @interface KtCoroutineException : KtKotlinThrowable
 - (instancetype)init __attribute__((swift_name("init()"))) __attribute__((objc_designated_initializer));
 + (instancetype)new __attribute__((availability(swift, unavailable, message="use object initializers instead")));
+
```
Comment (SvyatoslavScherbina) [inline kotlin-native/backend.native/tests/objcexport/expectedLazy.h:128]: What does cause these blank lines?

File: kotlin-native/backend.native/tests/objcexport/expectedLazy.h [hunk 2]
```diff
@@ -518,6 +532,36 @@ __attribute__((swift_name("TestGH3992.B")))
...
+/** @property xyzzy KDoc for foo? */
+@property (readonly) NSString *foo __attribute__((swift_name("foo")));
+
+/** @property foo KDoc for yxxyz? */
+@property int32_t yxxyz __attribute__((swift_name("yxxyz")));
+
+/**
+     * @param xyzzy is documented.
+     * This is multi-line KDoc
+     */
```
Comment (SvyatoslavScherbina) [inline kotlin-native/backend.native/tests/objcexport/expectedLazy.h:561]: Indents seem awkward. Is this what the frontend provides in PSI?

File: kotlin-native/backend.native/tests/objcexport/kdocExport.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    val xyzzy: String) {
+
+    /** Non-primary ctor KDoc: */
+    constructor() : this("blabla")
+
+    /** @property xyzzy KDoc for foo? */
+    val foo = "foo"
+    /** @property foo KDoc for yxxyz? */
+    var yxxyz = 0;
+}
```
Comment (SvyatoslavScherbina) [inline kotlin-native/backend.native/tests/objcexport/kdocExport.kt:29]: No tests for cases when StubRenderer has to emit both commits -- kdoc and `stub.comment`.

**Comment by knebekaizer** [comment]
Moved to another branch #4249

**Pull request closed without merging**